crates/aptos/src/move_tool/mod.rs

```diff
@@ -40,13 +52,100 @@ impl MoveTool {
     pub async fn execute(self) -> CliResult {
         match self {
             MoveTool::Compile(tool) => tool.execute_serialized().await,
+            MoveTool::Init(tool) => tool.execute_serialized_success().await,
             MoveTool::Publish(tool) => tool.execute_serialized().await,
             MoveTool::Run(tool) => tool.execute_serialized().await,
             MoveTool::Test(tool) => tool.execute_serialized().await,
         }
     }
 }
 
+/// Creates a new Move package at the given location
+#[derive(Parser)]
+pub struct InitPackage {
+    /// Name of the new move package
+    #[clap(long)]
+    name: String,
+    /// Path to create the new move package
+    #[clap(long, parse(from_os_str))]
+    package_dir: PathBuf,
```

Comment:
It's wierd that it requires both `--name` and `--package-dir`, can't it infer `--package-dir` from name?

Reply:
Name is the name of the package (in the move.toml, and differs from the path so it can't be inferred.  My example I made up a path, but the package is called something different

Reply:
Still, simplest case would be to just create a package with the same name for the directory
```
aptos move init --name mypackage
```
(or, in positional `--name` just `aptos move init mypackage`)

which would create `mypackage/sources` `mypackage/Move.toml`
```toml
[package]
name = "mypackage"
```
And if user would need to separate two, they would be able to specify `--package-dir` explicitly. 

I'd argue that it's better because there are fewer things to think about for the beginner. Oh, and also it would be the same as in Cargo
```
cargo new mypackage
```